packages/manager/apps/catalog/e2e/FilterAndSearch.e2e.ts

```diff
@@ -70,46 +60,67 @@ const validateProductCategories = async (
 test('should filter results based on Bare Metal Cloud universe', async ({
   page,
 }) => {
-  await clickButtonByName(page, 'filterButton');
-  await clickCheckboxByName(page, 'checkbox-universe-BareMetalCloud');
-  await clickButtonByName(page, 'applyFilterButton');
+  await clickButtonByText(
+    page,
+    searchTranslation.manager_catalog_search_filter_button,
+  );
+  await clickCheckboxByName(page, 'checkbox-universe-Bare_Metal_Cloud');
+  await clickButtonByText(
+    page,
+    filterTranslation.manager_catalog_filters_button_apply,
+  );
 
   const products = await waitForProducts(page, 5);
   await validateProductCategories(products, [
     'Dedicated Servers',
     'Virtual Private Servers',
     'Managed Bare Metal',
-    'Storage and Backup (BMC)',
-    'Storage and Backup (BMC)',
+    'Storage and Backup',
+    'Storage and Backup',
   ]);
 });
 
 test('should clear all filters when Clear All button is clicked', async ({
   page,
 }) => {
-  await clickButtonByName(page, 'filterButton');
-  await clickCheckboxByName(page, 'checkbox-universe-BareMetalCloud');
-  await clickButtonByName(page, 'applyFilterButton');
+  await clickButtonByText(
+    page,
+    searchTranslation.manager_catalog_search_filter_button,
+  );
+  await clickCheckboxByName(page, 'checkbox-universe-Bare_Metal_Cloud');
+  await clickButtonByText(
+    page,
+    filterTranslation.manager_catalog_filters_button_apply,
+  );
 
   await waitForProducts(page, 5);
 
-  await clickButtonByName(page, 'filterButton');
+  await clickButtonByText(
+    page,
+    searchTranslation.manager_catalog_search_filter_button,
+  );
 
   const links = await page.$$('osds-link');
   const resetFilterLink = links[0];
   if (resetFilterLink) await resetFilterLink.click();
 
-  await waitForProducts(page, 40);
+  await waitForProducts(page, 41);
```

Comment:
I think we should mock the call if it's not done : The goal is not to test the catalog API

Reply:
@aboungnaseng-ovhcloud If you can create a ticket about mocking data test for catalog pretty please.
i agree with @chipp972 on this point but just gonna scope it on another ticket.